kubernetes/annotations_and_labels.md

```diff
@@ -205,6 +206,14 @@ and should be placed inside the chart's `templates/` subdirectory and they
 define a couple of variables that can then be re-used throughout other
 templates in the chart.
 
+The team label is set using the `maintainers` field in Chart.yaml. The prefix
+`team-` identifies it and is used to set `.Values.maintainers.team`.
```

Comment:
I am not sure why opt to use a prefix instead of a key in a yaml map? A prefix means more parsing logic.

Reply:
We'd like to use the maintainers field in Chart.yaml but it only supports name, email and url AFAIK.

https://helm.sh/docs/topics/charts/

Reply:
What you mean by that is, you want to avoid presenting it like, `name: batman` but `name: team-batman`, right?

Reply:
@rossf7 From the docs
> Other fields will be silently ignored.

I also checked the [schema used by Chart Testing (`ct`)](https://github.com/helm/chart-testing/blob/master/etc/chart_schema.yaml), `maintainers` is optional, but if it's set then `name` is required. We could include a `team` key in `maintainers`, this will be ignored by linting and Helm. But we would still need to fill the `name` key... So yeh, I am not 100% sure, I would still prefer using a `team` key but not sure if it really matters if we have to complete `name` anyways.

Reply:
@rossf7 From the docs

Other fields will be silently ignored.

I also checked the schema used by Chart Testing (ct), maintainers is optional, but if it's set then name is required. We could include a team key in maintainers, this will be ignored by linting and Helm. But we would still need to fill the name key... So yeh, I am not 100% sure, I would still prefer using a team key but not sure if it really matters if we have to complete name anyways.